From f87552a85ec055276a7b7692782101bc2b705f72 Mon Sep 17 00:00:00 2001
From: Robert Marko <robert.marko@sartura.hr>
Date: Fri, 9 Jan 2026 13:27:00 +0100
Subject: [PATCH 69/73] dt-bindings: arm: AT91: document Novarq Tactical 1000

Novarq Tactical 1000 is a Microchip LAN9696 based 24x1G + 4x10G SFP switch.

Signed-off-by: Robert Marko <robert.marko@sartura.hr>
(cherry picked from commit 69b6f0b134664fcf39d7b94bbb4a689ef0af2c5d)
---
 Documentation/devicetree/bindings/arm/atmel-at91.yaml | 6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/Documentation/devicetree/bindings/arm/atmel-at91.yaml b/Documentation/devicetree/bindings/arm/atmel-at91.yaml
index 3a34b7a2e8d4..49666e92c04a 100644
--- a/Documentation/devicetree/bindings/arm/atmel-at91.yaml
+++ b/Documentation/devicetree/bindings/arm/atmel-at91.yaml
@@ -241,6 +241,12 @@ properties:
           - const: microchip,lan9668
           - const: microchip,lan966
 
+      - description: Novarq Tactical 1000
+        items:
+          - const: novarq,tactical-1000
+          - const: microchip,lan9696
+          - const: microchip,lan9691
+
       - description: Kontron KSwitch D10 MMT series
         items:
           - enum:
